### ISSUE TITLE
bug in normalizing scores

### ISSUE THREAD

**Issue by pbmstrk** [pbmstrk/QAKBC#4]
Currently the `doc_scores` are normalized as,

```
docscores = normalize(np.array(docscores)[:, np.newaxis], axis=0)[:, np.newaxis]
```

The `normalize` function is taken from `sklearn.preprocessing` and does not work as intended.

Instead use 

```
def normalize(arr):
    return arr/arr.sum(axis=0,keepdims=True)
```

**Comment by pbmstrk** [comment]
changed model - normalising scores is no longer necessary
